tools/memleak.py

```diff
@@ -135,6 +135,8 @@ def run_command_get_pid(command):
         help="capture only allocations larger than this size")
 parser.add_argument("-Z", "--max-size", type=int,
         help="capture only allocations smaller than this size")
+parser.add_argument("-O", "--obj", type=str, default="c",
+        help="attach to malloc & free in specified object instead of default")
```

Comment:
Sure, went for shorter help to not exceed 80 chars. Now looking at it "instead of default" does not seem to be necessary there, so maybe just "attach to malloc & free in the specified object"?